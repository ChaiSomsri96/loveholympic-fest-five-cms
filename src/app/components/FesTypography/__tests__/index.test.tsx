import * as React from 'react';
import { render } from '@testing-library/react';

import { FesTypography } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<FesTypography  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<FesTypography />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
