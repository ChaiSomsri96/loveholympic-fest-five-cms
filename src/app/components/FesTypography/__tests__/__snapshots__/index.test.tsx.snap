// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FesTypography  /> should match snapshot 1`] = `
.c0 {
  font-family: Roboto;
  font-weight: bold;
  font-size: 18px;
  line-height: 23px;
  margin-bottom: 5px;
  color: #657d9d;
}

<span
  class="c0"
>
  undefined
</span>
`;
